app/Relations/HasManySizeVariants.php

```diff
@@ -39,16 +39,7 @@ public function __construct(Photo $owningPhoto)
 	/**
 	 * Get the results of the relationship.
 	 *
-	 * @internal The parent class allows to return `mixed`, hence it is
```

Comment:
I would be happy if this comment was kept as it explains why this is happenning, because I do expect this error to come back with phpstan. If not we can always remove it later. :)